text/0000-lock-without-registry.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: 2017-05-09
+- RFC PR:
+- Yarn Issue:
+
+# Summary
+
+The lockfile yarn.lock should not include the base registry (`https://registry.npmjs.org`).
+
+# Motivation
+
+In yarn.lock, the `resolved` field includes registry such as `https://registry.npmjs.org`.
+In China, most developers will set it to `https://registry.npm.taobao.org` for speed; but it seems slow for travis-ci and circleci.
+
+# Detailed design
+
+Replace the `resolved` by a `hash` field.
```

Comment:
For my benefit:

_registry URL_:
* Looked up in `.yarnrc` and `.npmrc` first.
* Fall back to `https://registry.yarnpkg.com`

_package path_ (yamljs/-/yamljs-):
* Based on registry URL spec (I assume there's a spec that defines the structure of this portion of the path).
* Uses package name which is already defined elsewhere in `yarn.lock`.

_version number_:
* Already defined in `version` field.

_file extension_:
* Is this always `tgz`?

_hash_:
* The only part of the resolution path not covered elsewhere.